base/Dockerfile

```diff
@@ -28,8 +33,9 @@ RUN apk add --update --no-cache \
     /var/cache/apk/* \
     /.cache
 
+COPY --from=gobuilder /go/bin/supercronic /usr/local/bin/
 COPY --chmod=755 entry.sh /entry.sh
-COPY --link requirements.txt /
```

Comment:
I'm not 100% sure if this needs to be put back @modem7 - Docker for Mac errors and wouldn't let me build